tests/end_to_end.rs

```diff
@@ -60,6 +61,33 @@ fn test_env() {
     assert_eq!(String::from("Hello, World!\n"), greeting)
 }
 
+#[test]
+fn test_cleanup() {
+    let root = TempDir::new().unwrap();
+
+    let store_path = output_of_default_job(
+        &root,
+        &PathBuf::from("tests/end_to_end/env_cleanup/rbt.roc"),
+    )
+    .unwrap();
+
+    let echoed_variables = std::fs::read_to_string(store_path.join("out")).unwrap();
+
+    let variables: HashSet<&str> = echoed_variables
+        .split_ascii_whitespace()
+        .map(|str| {
+            *str.split_terminator('=')
+                .collect::<Vec<&str>>()
+                .first()
+                .unwrap()
+        })
+        .collect();
+
+    let expected_variables = HashSet::from(["HOME", "_", "PWD", "SHLVL"]);
```

Comment:
Other than HOME, are these bash builtins? Would we be able to eliminate them by using sh or calling the env binary directly? (I think it's a binary anyway)

Reply:
Yup, these are bash builtins. I'll try with sh/env, ty!

Reply:
Looks like `sh` (and most, if not all, other shells) also have these as builtins 😞 
I've tried calling `env` directly, but I'm not sure what to do with the value, since we need to save it in a file. Any suggestion?

Reply:
I think eventually it would probably be good to add an option to save a command's stdout. Otherwise we're gonna have to use shells a lot just to capture that. That said, I'm not sure how common this will be outside of tests! Do you have any intuition here?

Reply:
yeah, tackling in a new issue is definitely a better idea. You're welcome to look into it if you want; just make an issue and I'll add it to the right project.

We can leave the extra variables until then, but a comment explaining why they exist might be nice!

Reply:
>We can leave the extra variables until then, but a comment explaining why they exist might be nice!

I will!